docs/source/user_manual/tools.rst

```diff
@@ -66,6 +66,58 @@ HighlightTool
 
 ImageInspectorTool
 ==================
+The `ImageInspectorTool` is designed to work with an `ImagePlot` renderer to
+display the data space coordinates and data value of the displayed 2D array.
+The tool is designed to be used in conjunction with the
+:class:`ImageInspectorOverlay`: the tool collects mouse position and data
```

Comment:
One (perhaps annoying) request: could you specify the full module path for this class (and the others below)? That way Sphinx will generate a link to the corresponding part of the API docs. This is a little tricky since Sphinx picks up the imports from the `api` module, so e.g. this identifier would become ``:class:`chaco.tools.api.ImageInspectorOverlay` `` (and not ``:class:`chaco.tools.image_inspector_tool.ImageInspectorOverlay` ``)

Reply:
Good idea: linking is more important than page beauty.